app/ptxboa_functions.py

```diff
@@ -7,8 +7,10 @@
 import plotly.graph_objects as go
 import streamlit as st
 
+from ptxboa.api import PtxboaAPI
 
-def calculate_results_single(api, settings):
+
+def calculate_results_single(api: PtxboaAPI, settings):
```

Comment:
This function only modifies setting keys and removes the prefix "sel" for all parameters and adds the suffix "name" to "country" and "res_gen". Is there a reason why backend and frontend use different keys? Maybe we can harmonize this some time (not in this PR).

Reply:
good idea. i created an issue to deal with this #27